README.md

```diff
@@ -10,7 +10,24 @@ The engine that allows editors to build on top of Jest.
 
 This is only useful if you are interested in building an editor integration for Jest.
 
-For now as an end user, we'd recommend looking at either [vscode-jest](https://github.com/jest-community/vscode-jest/) or [majestic](https://github.com/Raathigesh/majestic/).
+For now as an end user, we'd recommend looking at either [vscode-jest](https://github.com/jest-community/vscode-jest/), [vscode-jest-runner](https://github.com/firsttris/vscode-jest-runner) or [majestic](https://github.com/Raathigesh/majestic/).
+
+## API
+```
+parse(   
+  filePath: string,   
+  serializedData?: string,   
+  strictMode: boolean = false,     
+)
+```
+Parse is a static Jest parser which uses Babel 7 and supports js,jsx,mjs,ts,tsx files.   
+
+[Supported ECMAScript proposals](https://github.com/babel/babel/blob/928b9f8c9518284eac6d0598633f2ec373fc6d0c/packages/babel-parser/typings/babel-parser.d.ts#L97)
+
+- filePath = Path to the file you want to parse.
+- serializedData = Serialized data, will be used instead of the filePath if available (optional).
+- strictMode = If this option is activated the parser throws an exception if the filetype is not detected, defaults to false.
+
```

Comment:
great way to start documenting the API 👍 , can we throw in an example, I am a big fan of "learn-by-doing'

Reply:
would be best to create an example with typescript and to link it in the readme (maybe follow up PR)